frontend/src/components/ComboboxInput.spec.ts

```diff
@@ -60,6 +60,18 @@ describe('Combobox Element', () => {
     expect(screen.queryByLabelText('dropdown-option')).not.toBeInTheDocument()
   })
 
+  it("should not show sideInformation 'amtlich'", async () => {
+    renderComponent()
+
+    const openComboboxContainer = screen.getByLabelText('Dropdown öffnen')
+    await user.click(openComboboxContainer)
+    await vi.advanceTimersByTimeAsync(debounceTimeout)
+    expect(screen.getAllByLabelText('dropdown-option')).toHaveLength(2)
+    // a plain .not.toBeInTheDocument() should be available at some time
+    // cf. https://github.com/vitest-dev/vitest/pull/6634
+    expect(() => screen.getByText('amtlich')).toThrowError()
```

Comment:
You could change it to
```suggestion
    expect(() => screen.queryByText('amtlich')).not.toBeInTheDocument()
```